src/domain/notifications/notifications.repository.interface.ts

```diff
@@ -14,4 +14,16 @@ export interface INotificationsRepository {
     device: Device,
     safeRegistration: SafeRegistration,
   ): Promise<SafeRegistration>;
+
+  /**
+   * Deletes a a notification target, identified by its client {@link uuid}.
+   * @param chainId
+   * @param uuid
+   * @param safeAddress
+   */
+  deleteDevice(
```

Comment:
nitpick: Maybe `unregisterDevice` is more meaningful. We have `deleteDeviceRegistration` on the API layer which I think is a good naming choice.

Reply:
Changed in [d880fcb](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/d880fcb3732f5bd60894507bcb3104e4a46b7714)